examples/flutter_gallery/lib/demo/text_field_demo.dart

```diff
@@ -30,11 +30,13 @@ class TextFieldDemoState extends State<TextFieldDemo> {
     ));
   }
 
+  bool _autovalidate = false;
   GlobalKey<FormState> _formKey = new GlobalKey<FormState>();
   GlobalKey<FormFieldState<InputValue>> _passwordFieldKey = new GlobalKey<FormFieldState<InputValue>>();
   void _handleSubmitted() {
     FormState form = _formKey.currentState;
-    if (form.hasErrors) {
+    _autovalidate = true;  // Start validating on every change.
```

Comment:
Since this code may serve as a cut-and-pasteable example maybe it would be better to only set _autovalidate true in the `!form.validate()` clause.